packages/core/src/core.ts

```diff
@@ -654,23 +650,8 @@ class ThresholdKey implements ITKey {
     delete1OutOf1?: boolean;
   } = {}): Promise<InitializeNewKeyResult> {
     let seed: Uint8Array;
-    if (this.keyType === KeyType.secp256k1) {
-      const tmpPriv = importedKey || generatePrivate(this.keyType);
-      this._setKey(new BN(tmpPriv));
-    } else {
-      seed = importedKey ? importedKey.toBuffer() : nacl.randomBytes(32);
-
-      const keyPair = nacl.sign.keyPair.fromSeed(seed);
-      // need to decode from le ??
-      const tempPriv = new BN(keyPair.secretKey.slice(0, 32)).umod(this.ecCurve.n);
-      this._setKey(tempPriv);
-
-      // testing and checking code - to remove
-      // const decMsg = await this.decrypt(encMsg);
-      // const decSeed = Buffer.from(decMsg).toString("hex");
-      // console.log("decSeed: ", decSeed);
-      // console.log("seed: ", seed);
-    }
+    const tmpPriv = importedKey || generatePrivate(this.keyType);
```

Comment:
we need the seed for export
with just ed25519 scalar, it could not be imported to other wallet like solana wallet

Reply:
understood, lets get this working and loop back to adding this in.